tests/service/data/SyntaxTree/Member/Static 01.fs.bsl

```diff
@@ -24,32 +24,20 @@ ImplFile
                                       MemberKind = Member },
                                SynValInfo ([[]], SynArgInfo ([], false, None)),
                                None),
-                            FromParseError (Wild (5,10--5,10), (5,10--5,10)),
-                            None,
-                            ArbitraryAfterError
-                              ("classDefnMember1", (5,10--5,10)), (5,4--5,10),
-                            NoneAtInvisible,
+                            Paren (Const (Unit, (7,7--7,9)), (7,7--7,9)), None,
+                            ArbitraryAfterError ("memberCore2", (7,9--7,9)),
+                            (7,7--7,9), NoneAtInvisible,
                             { LeadingKeyword = Static (5,4--5,10)
                               InlineKeyword = None
-                              EqualsRange = None }), (5,4--5,10));
-                      LetBindings
```

Comment:
@T-Gro This is what I was afraid of: this and the following test cases got completely broken. 😞